### DISCUSSION TITLE
Just wondering does FLTK now support Android and iOS?

### DISCUSSION THREAD

**Discussion by Raj2032** [fltk/fltk#1034] [General]
I would like to _write once and run everywhere_. If I designed something for Linux then I would like to port this over to Android. Just wondering if this is possible or not at this stage?

**Comment by Albrecht-S** [comment]
Currently neither `Android` nor `iOS` are supported FLTK platforms.

There has been a partial Android port but it was removed because it was hard to maintain and didn't get ready in time. There may be another try later but there's no schedule yet.

Porting to iOS has been considered but will very likely not happen, IIRC because of the way Apple checks and allows/disallows programs in their Appstore. Others may know better about the Apple/iOS status. As far as I can tell there has not been any attempts to port FLTK to iOS yet.

FTR: commit b275ff07158e80d1744ddb2f6c51094a87cf079a (`git checkout experimental-2022-01`) is the last commit before experimental code like the Android port has been removed. You may want to try this but it's meanwhile more than 2.5 years old and there will be no support.

**Comment by Raj2032** [comment]
I see thanks mate :)
